src/SFA.DAS.EmployerUsers.Application/Events/AccountLocked/GenerateAndEmailAccountLockedEmailHandler.cs

```diff
@@ -109,14 +109,20 @@ public async Task Handle(AccountLockedEvent notification)
             }
         }
 
+        private async Task<string> GetEmployerAccountsBaseUrl()
+        {
+            return (await GetConfig())?.EmployerAccountsBaseUrl;
+        }
+
         private async Task<string> GenerateCode()
         {
-            var codeLength = (await GetConfig())?.UnlockCodeLength ?? 6;
+            var codeLength = (await GetConfig())?.Account?.UnlockCodeLength ?? 6;
             return _codeGenerator.GenerateAlphaNumeric(codeLength);
-        }
-        private async Task<AccountConfiguration> GetConfig()
+        }      
+
+        private async Task<EmployerUsersConfiguration> GetConfig()
         {
-            return (await _configurationService.GetAsync<EmployerUsersConfiguration>())?.Account;
+            return await _configurationService.GetAsync<EmployerUsersConfiguration>();

```

Comment:
`EmployerUsersCofiguration` is already registered in the IoC. Should be able to simply add it to the `GenerateAndEmailAccountLockedEmailHandler.cs` constructor.

Should be able to remove the service and private methods.